src/session.ts

```diff
@@ -1,19 +1,46 @@
 import { NodeDriverServiceProvider } from "@mongosh/service-provider-node-driver";
 import { ApiClient } from "./common/atlas/apiClient.js";
 import config from "./config.js";
+import { Implementation } from "@modelcontextprotocol/sdk/types.js";
 
 export class Session {
+    sessionId?: string;
     serviceProvider?: NodeDriverServiceProvider;
-    apiClient?: ApiClient;
+    apiClient: ApiClient;
+    agentClientName?: string;
+    agentClientVersion?: string;
```

Comment:
```suggestion
    agentRunner?: {
    	name: string;
   	version: string;
   }
```
(can also be agentClient, that makes sense; I just find the agent's client to possibly be the user?)